modules/solace/solace.go

```diff
@@ -35,14 +35,32 @@ func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustom
 		}
 	}
 
+	// Build exposed ports from services
+	var exposedPorts []string
+	for _, service := range settings.services {
+		exposedPorts = append(exposedPorts, fmt.Sprintf("%d/tcp", service.Port))
+	}
+
+	// Build wait strategies - combine the default wait strategy with port-based wait strategies for each service
+	var waitStrategies []wait.Strategy
```

Comment:
suggestion: pre-allocate the size of the slice, as we know it's len(settings.services + 1)

Reply:
done.